packages/angular/build/src/utils/server-rendering/manifest.ts

```diff
@@ -69,18 +69,22 @@ export function generateAngularServerAppEngineManifest(
       const end = localeWithBaseHref[localeWithBaseHref.length - 1] === '/' ? -1 : undefined;
       localeWithBaseHref = localeWithBaseHref.slice(start, end);
 
-      entryPointsContent.push(`['${localeWithBaseHref}', () => import('${importPath}')]`);
+      entryPoints[localeWithBaseHref] = `() => import('${importPath}')`;
     }
   } else {
-    entryPointsContent.push(`['', () => import('./${MAIN_SERVER_OUTPUT_FILENAME}')]`);
+    entryPoints[''] = `() => import('./${MAIN_SERVER_OUTPUT_FILENAME}')`;
   }
 
   const manifestContent = `
 export default {
   basePath: '${baseHref ?? '/'}',
-  entryPoints: new Map([${entryPointsContent.join(', \n')}]),
+  entryPoints: {
+    ${Object.entries(entryPoints)
+      .map(([key, value]) => `'${key}': ${value}`)
```

Comment:
Nit: I probably should have noticed this the first time, but do we need to escape `key` if it contains a single quote? I guess it would be pretty rare for an entry point to contain a single quote in the file name. Are we comfortable relying on that assumption?

Reply:
In this case the is the locale, thus cannot contain non valid letters.